### PR TITLE
🌐 Sync Public and Master

### PR DESCRIPTION
**Pull request by dmiller15**
## Description

Sync prod joint genotyping wf and cavatica public app.

Resolves https://github.com/d3b-center/bixu-tracker/issues/592 
Includes estimates for https://github.com/d3b-center/bixu-tracker/issues/627

## Type of change

- [x] Bug fix (Fixes Joint Genotyping ouptut)
- [x] New feature (New input handling)
- [x] Breaking change (changes to the public app)
- [x] Documentation

## How Has This Been Tested?

- [x] Joint Genotyping Test: https://cavatica.sbgenomics.com/u/kfdrc-harmonization/sd-46sk55a3-03/tasks/9b179530-5f1b-43e2-9a8d-19258bc81ee6/
- [x] Single Sample Test: https://cavatica.sbgenomics.com/u/kfdrc-harmonization/sd-bhjxbdqk-01/tasks/0cc9a14c-cb05-4991-94d9-95baac9992cc/#
- [x] Tool passes cwltool validation
- [x] Tool passes Rabix validation

**Test Configuration**:
* Environment: Cavatica, Rabix 1.0.5, cwltool 3.0.20200324120055
* Test files: See Cavatica test

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: tools/picard_collectaggregationmetrics.cwl [hunk 1]
```diff
@@ -0,0 +1,36 @@
+cwlVersion: v1.0
+class: CommandLineTool
+id: picard_collectaggregationmetrics
```
Comment (migbro) [inline tools/picard_collectaggregationmetrics.cwl:3]: Is this tool supposed to be in this repo?!

File: tools/kfdrc_peddy_tool.cwl [hunk 2]
```diff
@@ -16,7 +16,7 @@ arguments:
       -m peddy
       -p 4
       --sites hg38
-      --prefix $(inputs.output_basename)
+      --prefix $(inputs.output_basename).peddy
```
Comment (migbro) [inline tools/kfdrc_peddy_tool.cwl:19]: hmmm, can see why you added, but then the peddy  output becomes peddy.peddy...I'm of two minds on this one

File: tools/picard_collectaggregationmetrics.cwl (hunk 1 above)
Comment (dmiller15) [reply tools/picard_collectaggregationmetrics.cwl:3]: Sneaky cwl found its way into the repo

File: tools/kfdrc_peddy_tool.cwl (hunk 2 above)
Comment (dmiller15) [reply tools/kfdrc_peddy_tool.cwl:19]: Git blame says you did it: https://github.com/kids-first/kf-jointgenotyping-workflow/commit/84d03071a71c363898594597490b97ed6db7d017

Comment (migbro) [reply tools/kfdrc_peddy_tool.cwl:19]: Someone should fire that guy! I'll agree with past self and let it be.

File: tools/picard_collectaggregationmetrics.cwl (hunk 1 above)
Comment (dmiller15) [reply tools/picard_collectaggregationmetrics.cwl]: Deleted

**Pull request merged**